OpenIPSL/Electrical/ThreePhase/Buses/MeasurementBus.mo

```diff
@@ -0,0 +1,81 @@
+within OpenIPSL.Electrical.ThreePhase.Buses;
+model MeasurementBus
+  extends ThreePhaseComponent;
+  OpenIPSL.Interfaces.PwPin p1
+    annotation (Placement(transformation(extent={{80,80},{100,100}})));
+  OpenIPSL.Interfaces.PwPin p2
+    annotation (Placement(transformation(extent={{80,-10},{100,10}})));
+  OpenIPSL.Interfaces.PwPin p3
+    annotation (Placement(transformation(extent={{80,-100},{100,-80}})));
+  OpenIPSL.Interfaces.PwPin p4
+    annotation (Placement(transformation(extent={{-100,80},{-80,100}})));
+  OpenIPSL.Interfaces.PwPin p5
+    annotation (Placement(transformation(extent={{-100,-10},{-80,10}})));
+  OpenIPSL.Interfaces.PwPin p6
+    annotation (Placement(transformation(extent={{-100,-100},{-80,-80}})));
+  SI.ActivePower Pa(displayUnit="MW") "Active Power suplyed by the Infinite bus (phase a)";
+  SI.ReactivePower Qa(displayUnit="Mvar") "Reactive Power suplyed by the Infinite bus (phase a)";
+  SI.ActivePower Pb(displayUnit="MW") "Active Power suplyed by the Infinite bus (phase b) (MW)";
+  SI.ReactivePower Qb(displayUnit="Mvar") "Reactive Power suplyed by the Infinite bus (phase b) (MVAr)";
+  SI.ActivePower Pc(displayUnit="MW") "Active Power suplyed by the Infinite bus (phase c) (MW)";
+  SI.ReactivePower Qc(displayUnit="Mvar") "Reactive Power suplyed by the Infinite bus (phase c) (MVAr)";
+  Modelica.Blocks.Interfaces.RealOutput pa = Pa/S_b annotation (Placement(transformation(
```

Comment:
@MaximeBaudette shouldn't it be Pa/S_p instead of Pa/S_b?

Reply:
That's a good question! 
This is a measurement in p.u., so having the measurements based on the system base makes sense to me... 
Or maybe to avoid confusion, the output should directly be in W / VAr ?

Reply:
hmm, so it makes sense that it is Pa/S_b. Maybe just add a description like "Measured active power in phase A (pu, system base)" to avoid any kind of confusion for users, what do you think? In addition, I don't know if @dietmarw prefers to have it like "[pu, system base]"

Reply:
Yes I would prefer units i brackets and definitely add a description string.

@lvanfretti  I think we are talking about `pa` which **is** the per-unit power. The issue was just the base was mistaken.

Reply:
Great, so please @MaximeBaudette just add the "Measured active power in phase A [pu, system base]" as a description of the output. After that the pull-request would be ready to be merged.